packages/babel-template/src/populate.js

```diff
@@ -15,7 +15,14 @@ export default function populatePlaceholders(
       if (
         !Object.prototype.hasOwnProperty.call(replacements, placeholder.name)
       ) {
-        throw new Error(`No substitution given for "${placeholder.name}"`);
+        throw new Error(`Error: No substitution given for "${
+          placeholder.name
+        }". If this is not meant to be a
+          placeholder you may want to consider passing one of the following options to babel-template:
```

Comment:
this formatting was put in by prettier.  maybe it expects `{` to always be the start of an object and thus need a line break? 

let me know if this 🆗

Reply:
Template literals, the ``, preserve spaces and new lines. You'll need to construct your string something like the way it's done here https://github.com/babel/babel/blob/180eda3211faa855654757bc44aeaf3961a8be05/packages/babel-core/src/transformation/normalize-file.js#L82

Otherwise, the error message will have unnecessary indentation.

Reply:
@maaz93 
It seems to keep complaining about that `{`
```shell
error: Replace `··`Error:·No·substitution·given·for·"${placeholder.name` with ``Error:·No·substitution·given·for·"${⏎············placeholder.name⏎··········` (prettier/prettier) at packages/babel-template/src/populate.js:19:11:
  17 |       ) {
  18 |         throw new Error(
> 19 |             `Error: No substitution given for "${placeholder.name}". If this is not meant ` +
     |           ^
  20 |               `to be a placeholder you may want to consider passing one of the following options to babel-template:` +
  21 |               `- { placeholderPattern: false, placeholderWhitelist: new Set(['${placeholder.name}'])}` +
  22 |               `- { placeholderPattern: /^${placeholder.name}$/ }`

```

Reply:
Alright. This seems a little tricky. I formatted it here to work fine:

```js
        const placeholderName = placeholder.name;
        throw new Error(
          `Error: No substitution given for "${placeholderName}". If this is not meant to be a placeholder ` +
            `you may want to consider passing one of the following options to babel-template:` +
            `- { placeholderPattern: false, placeholderWhitelist: new Set(['${placeholderName}'])}` +
            `- { placeholderPattern: /^${placeholderName}$/ }`,
        );
```
It was complaining about `printWidth` and the `{` placement. I've overcome the `{` by not having a property access, rather storing it in `placeholderName`. Also, I changed number of characters per line in one of the lines to meet the allowed `printWidth`. Let me know if this works